src/main/resources/application.yml

```diff
@@ -110,10 +110,10 @@ delius-integration-context:
       nsi-mapping:
         nsi-status: INPROG
         service-category-to-nsi-type:
-          "[Accommodation]": CRS01
-          "[Education, Training and Employment (ETE)]": CRS02
-          "[Finance, Benefits and Debt (FBD)]": CRS03
-          "[Dependency and Recovery (D&R)]": CRS04
+          428ee70f-3001-4399-95a6-ad25eaaede16: CRS01
```

Comment:
With this change does the endpoint now accept the key, so do clients send `428ee70f-3001-4399-95a6-ad25eaaede16` in the message? I'm struggling to understand why the key isn't just CRS03 or whatever?

Reply:
Interventions has no knowledge of nsi types (e.g.CRS03) - it's a Delius specific concept. Interventions deals with service categories.

Reply:
So still confused where the actual keys are referenced, are the sent in the body of the request?

Reply:
Yes it is sent in the request from interventions

Reply:
Thanks, got it!